### PR TITLE
[TE] rootcause poc - support multiple different display modes for timeseries

### PR DESCRIPTION
**Pull request by apucher**
* modes for absolute, relative, log

* smooth transition without unloading

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 1]
```diff
@@ -127,8 +135,8 @@ export default Ember.Component.extend({
     'selectedUrns',
     'analysisRange',
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: does this still need to depend on 'analysisRange' ?

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 2]
```diff
@@ -219,6 +231,31 @@ export default Ember.Component.extend({
     }
   },
 
+  _transformSeries(mode, series) {
+    switch(mode) {
+      case 'absolute':
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: let's save the string `absolute`, `relative` and `log` into a const obj TYPE
and refer to them as TYPE.absolute, TYPE.relative and TYPE.log

https://github.com/linkedin/pinot/pull/2081/files#diff-ba88a3a55f4254f597817fd4301d5280R20 can also be updated accordingly

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 3]
```diff
@@ -219,6 +231,31 @@ export default Ember.Component.extend({
...
+      case 'relative':
+        return this._transformSeriesRelative(series);
+      case 'log':
+        return this._transformSeriesLog(series);
+    }
+    return series;
+  },
+
+  _transformSeriesRelative(series) {
+    const first = series.values.filter(v => v)[0];
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: Let's try using better variable names that just letters for readability. The compiler would shorten them anyway for performance reason.

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: what would be a readable suggestion here? :D

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js (hunk 1 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: fixed

**Comment by ttbach** [comment]
I like the short PRs. Keep them coming!

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js (hunk 2 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: fixed

**Pull request merged**